x/ibc/03-connection/keeper/handshake_test.go

```diff
@@ -115,7 +115,7 @@ func (suite *KeeperTestSuite) TestConnOpenTry() {
 			connectionKey := ibctypes.KeyConnection(testConnectionIDA)
 			proofInit, proofHeight := queryProof(suite.chainA, connectionKey)
 
-			consensusKey := ibctypes.KeyConsensusState(testClientIDB, consensusHeight)
+			consensusKey := prefixedClientKey(testClientIDB, ibctypes.KeyConsensusState(uint64(consensusHeight)))
```

Comment:
Currently using this helper function rather than creating a prefix and using `ApplyPrefix`